packages/x-date-pickers/src/internals/components/PickerFieldUI.tsx

```diff
@@ -313,16 +310,6 @@ export interface PickerFieldUIProps<
   TEnableAccessibleFieldDOMStructure extends boolean,
   TProps extends UseFieldProps<TEnableAccessibleFieldDOMStructure>,
 > {
-  /**
-   * Overridable component slots.
-   * @default {}
-   */
-  slots?: PickerFieldUISlots;
```

Comment:
This is the main change, `PickerFieldUi` no longer supports `slots` and `slotProps` AND the same props in the context.
In every case, these props are passed via context.